### ISSUE TITLE
rum can't capture video type transactions

### ISSUE THREAD

**Issue by LHB6540** [elastic/apm-agent-rum-js#1417]
In my site html site I try to follow this [documentation](https://www.elastic.co/guide/en/apm/agent/rum-js/5.x/install-the-agent.html) Synchronous / Blocking Pattern and Asynchronous / Non-Blocking Pattern in the two ways to install the agent. But no matter which one, I can capture image, css and other types of transactions, but I can't capture video access or loading transactions.
My video reference code is as follows, it is located in html->body, the same location as those image type files
```
<video id="bgVideo" x5-video-player-fullscreen="true" x5-playsinline
     playsinline webkit-playsinline muted autoplay loop poster="/assets/images/m/common/kv.jpg">
     <source src="https://xxx.com/landing/dhs/mbg77.mp4?v=1" type="video/mp4">
</video>
```
I would like to know if the capture of video type is supported, if so, how should I troubleshoot and solve the current problem

**Comment by devcorpio** [comment]
Hi @LHB6540,

I have a few questions:

- What do you mean by capturing video type transactions,  could you elaborate more? 
- What are you exactly expecting to capture and see in Kibana UI?

Cheers,
Alberto

**Comment by LHB6540** [comment]
> 
It means that I want to see the video request or loading time in kibana's tracking ui.
As shown in the picture below, I can observe the loading time of the referenced picture, but there is no loading time of the video type file, which means that no video transaction event is reported. Even if I use the Synchronous / Blocking Pattern to ensure that rum.js is completed before the video loads, the result is still like this。
![image](https://github.com/elastic/apm-agent-rum-js/assets/41095303/62e418b0-abec-476b-b8ea-da28bd920a47)

**Comment by devcorpio** [comment]
I see. thanks for the clarification

Let's check if the resource timing API is identifying the video resource properly:

Steps:

1. Load your page
2. Once the page is loaded, go to your browser console and execute:
   `performance.getEntriesByType("resource").filter(entry => entry.initiatorType === 'video')`

Let me know if it's returning any entry and if the values are that ones that you would expect to see in Kibana

thanks,
Alberto

**Comment by devcorpio** [comment]
You are right @LHB6540. When we developed this feature a while ago it was decided not to include all the resources available. So as you said, we are not capturing resources such as video (audio and embed either)

The list below contains the resources that we are currently capturing:

```
  link
  css
  script
  img
  xmlhttprequest
  fetch
  beacon
  iframe
```

will discuss this with the team and we'll let you know once we have more info

Thanks,
Alberto

**Comment by LHB6540** [comment]
OK,thank you
